src/Microsoft.DotNet.Build.Tasks.Feed/Models/IndexJson.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NuGet.Versioning;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Build.Tasks.Feed
+{
+    public class IndexJson
+    {
+        public List<string> versions { get; set; }
```

Comment:
nit: Should be capital V for public Properties.

Reply:
removed.